libtalek/subscription.go

```diff
@@ -74,13 +74,13 @@ func (s *Subscription) generatePoll(config *ClientConfig, seqNo uint64) (*common
 	// The first Trust domain is the one with the explicit bucket bit-flip.
 	k0, k1 = s.Seed2.KeyUint128()
 	bucket2 := siphash.Hash(k0, k1, seqNoBytes) % config.CommonConfig.NumBuckets
-	args[1].TD[0].RequestVector = make([]byte, config.CommonConfig.NumBuckets/8+1)
-	args[1].TD[0].RequestVector[bucket2/8] |= 1 << (bucket1 % 8)
+	args[1].TD[0].RequestVector = make([]byte, (config.CommonConfig.NumBuckets+7)/8)
+	args[1].TD[0].RequestVector[bucket2/8] |= 1 << (bucket2 % 8)
```

Comment:
This was a bug. this was the part copied from the args[0] which is for bucket1 copied down to now be for args[1] (bucket2), but one instance hadn't been changed yet to bucket2.